infra/deploy_kubernetes_manifests.tf

```diff
@@ -104,6 +103,9 @@ resource "kubernetes_service_account" "kubernetes_manifests_deployer_service_acc
       "iam.gke.io/gcp-service-account" = google_service_account.kubernetes_manifests_deployer_service_account.email
     }
   }
+  depends_on = [
+    google_container_cluster.my_cluster_config
+  ]
```

Comment:
This `depends_on` was added in hopes to fix an issue that Drew encountered when running `terraform apply`:
```
│ Error: Unauthorized
│
│   with kubernetes_service_account.kubernetes_manifests_deployer_service_account,
│   on deploy_kubernetes_manifests.tf line 99, in resource "kubernetes_service_account" "kubernetes_manifests_deployer_service_account":
│   99: resource "kubernetes_service_account" "kubernetes_manifests_deployer_service_account" {

```

Reply:
It might not be the right fix.